Chutzpah/Models/ChutzpahTestSettingsFile.cs

```diff
@@ -350,35 +356,40 @@ public ChutzpahTestSettingsFile InheritFrom(ChutzpahTestSettingsFile parent)
             this.CodeCoverageIgnores = parent.CodeCoverageIgnores.Concat(this.CodeCoverageIgnores).ToList();
             this.Transforms = parent.Transforms.Concat(this.Transforms).ToList();
 
+            foreach (var browserArgument in parent.BrowserArguments)
+            {
+                this.BrowserArguments[browserArgument.Key] = browserArgument.Value;

```

Comment:
The idea is that when you inherit from a parent chutzpah.json file you take the parent's value unless you defined it